app/controllers/caseload_controller.rb

```diff
@@ -20,18 +22,7 @@ def new
 private
 
   def sort_allocations(allocations)
-    if params['sort'].present?
-      sort_field, sort_direction = params['sort'].split.map(&:to_sym)
-    else
-      sort_field = :last_name
-      sort_direction = :asc
-    end
-
-    # cope with nil values by sorting using to_s - only dates and strings in these fields
-    allocations = allocations.sort_by { |sentence| sentence.public_send(sort_field).to_s }
-    allocations.reverse! if sort_direction == :desc
-
-    allocations
+    sort_collection(allocations, :last_name)
```

Comment:
It would be nice for `default_sort` and `default_direction` to be keyword parameters, to make it more obvious what their meaning is.

Otherwise this line of code reads like we're always sorting by `:last_name`. The fact that we actually read the request `params`, and `:last_name` is just a fallback, is hidden within the implementation of `Sorting` concern.

```suggestion
    sort_collection(allocations, default_sort: :last_name)
```

(where `default_direction` is still optional and `:asc` if not given)